experiments/dataSources/worldBank/readme.md

```diff
@@ -0,0 +1,102 @@
+
+#### Create a CORS entry
+
+* Allowing the browser to retrieve data directly is the fastest method for data sources (as opposed to using the Server Proxy), but if you are using Skuid on Salesforce, you will be required that you whitelist the endpoint so you don't have a CORS violation.
+* Within Salesforce Setup, go to **Security > CORS**.
+* Create a new entry - generally it will be the same domain where you ultimately intend to host the page. ``https://<<mydomain>>.visual.force.com``
+
+### Create a page with this REST data source to review World Bank statistics
+
+- Create a new page
+
+- Create a model that returns GDP data for all countries.  
+    - Data Source Type: REST
+    - Data Source: WorldBank
+    - Model Behavior: Read-Only
+    - Data Source URL: ``countries/ALL/indicators/SP.POP.TOTL?format=json``
+    - All other properties can be default. 
+
+NOTE about Path to Contents:  Because WorldBank puts nests in JSON - the root is just metadata about the call.  In order to get tot he data iteslf, you have to go into the XML and indicate that the path to contents = ``1``.   See the example xml below:  (See final attribute of node) 
+
+``<model id="Sources" query="true" createrowifnonefound="false" datasource="WorldBank" verb="GET" payloadcontenttype="JSON" endpoint="/sources?format=json" pathtocontent="1">``
+
+Once you have done this in XML you can go back to the composer. (Once you have one model set up this way,  clone the model to set up the next one and you will not have to go back to the XML to set "path to contents")
+
+- Select "Fields" on the model properties to add the following fields to your model: 
+    - countryiso3code
+    - date
+    - value
+
+
+## Page Ideas
+
+- Add a table to your page to quickly see the data being returned. 
+- Use a chart to see data changing over time
+- Replace the ``ALL`` value in the Data Source URL with ``{{Country}}`` - and use another model to populate that merge value with another ISO-3 Country value.  (BRA for Brasil). 
+- Create a "countries" model  (This API call will return countries:  ``http://api.worldbank.org/v2/country?format=json ``) and then use an action sequence to pass a particular row value into the merge syntax in the Data source above. 
+
+
+## Sample Page
+
+Two sample pages are included in this experiment.  Create a new V2 Page and copy the XML from these pages for more fully constructed examples. 
+
+### WorldBankSources
+
+- This page provides an index of all the statistical measures provided by the World Bank - along with the URL you'd need to use to create a model for this measure, and a sample dataset queried for this measure.
+
+### WorldBankDashboard
+
+- This page allows the user to select a single country (from a statstical region) and then shows several tabs of statical measures about the country. 
```

Comment:
This seems a little buggy. Several regions I select offer no countries, and thus I can't do anything on the page. For example, this option—Africa—doesn't allow me to do anything. Is this intended behavior?

Reply:
Other seem to have odd behaviors, like `Carribean small states > Bahamas, The` displays nothing on the Poverty and Inequality tabs. Other tabs work fine, even the `Inequality` tab which has a notice: "No data to display." Is that notice on the first tab as well?

Reply:
After selecting a Country for the first time, selecting a different country breaks the page:

![Screen Shot 2020-02-26 at 1 34 10 PM](https://user-images.githubusercontent.com/7629304/75375761-bf8c3200-589c-11ea-8226-bed4d60f8743.png)

Reply:
Some Regions do not return country values.  I added some client side conditions to remove regions that don't return countries,  I will add some more.

Reply:
That makes sense. Since we don't know all of them at this time, I'd consider putting a note in this README that describes your client-side condition work around.